### PR TITLE
PSU Examples

### PR DESCRIPTION
**Pull request by ruthtillman**
4 sample records from PSU.

### CODE REVIEW

File: _examples/psu/esther-lawton-interview-cassette.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Esther Lawton: oral history cassette
```
Comment (anarchivist) [inline _examples/psu/esther-lawton-interview-cassette.json]: Values with colons need to be wrapped in quotes.

File: _examples/psu/esther-lawton-interview-digital.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: Esther Lawton: oral history Digital Object
```
Comment (anarchivist) [inline _examples/psu/esther-lawton-interview-digital.json]: Values with colons need to be wrapped in quotes.

File: _examples/psu/united-mine-workers-safety-division.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
...director in February 1972, during his transition from president of District 31 to UMW vice-president. On 16 October 1972, Kenneth F. Wells was appointed director. In 1973, as a component of an overhaul of the UMW organizational structure under President Arnold Miller, the Safety Division was expanded to include more regional inspectors, as well as an engineer and a lawyer devoted solely to safety issues.","This collection documents one of the overriding objectives of the union: safety for miners. An alphabetical run of subject files and correspondence concerns health and safety regulations, reports, contractual agreements, petitions for modifications of mandated safety standards, and training programs. A reference library includes reports and publications on workmen’s compensation and on health and safety hazards associated with coal mining, including black lung disease, potentially dangerous chemicals, high levels of noise, workmen’s compensation, and other issues. Also in the reference library are transcripts from hearings on mine disasters, and a run of the Mine and Safety Reporter newsletter. There are photographic slides and seven 16-mm films, all used for training programs."]
```
Comment (anarchivist) [inline _examples/psu/united-mine-workers-safety-division.json]: This line is missing a trailing comma.

Comment (ruthtillman) [reply _examples/psu/united-mine-workers-safety-division.json]: thanks @anarchivist .

**Comment by ruthtillman** [comment]
@anarchivist 💯 thanks much!

(also belatedly figured out I set my notifications to my ND account so... that explains a lot about what I wasn't getting, sorted.)

**Pull request merged**